website/docs/releases/2024/v2024.8.md

```diff
@@ -112,8 +112,4409 @@ helm upgrade authentik authentik/authentik -f values.yaml --version ^xxxx.x
+ - web: add  HTMLTagNameElementMaps to everything to activate lit analyzer (#10217)
+ - web: all aboard the anti-if bus, according to tooling (#10220)
+ - web: bug  - licenseStatus is not defined on initial render (#10894)
+ - web: disable reading dark mode out of the UI by default (#10256)
+ - web: enable custom-element-manifest and DOM/JS integration checking. (#10177)
+ - web: enhance search select with portal, overflow, and keyboard controls (#9517)
+ - web: fix bad name target that's breaking build (#10506)
+ - web: fix dark theme and theme switch (#10667)
+ - web: fix dark theme and theme switch (#10667)
+ - web: fix docker build for non-release versions (#10154)
+ - web: fix early modal stack depletion (#10068)
+ - web: fix lint (#10524)
+ - web: fix mismatched button labels for boundpolicy and boundstage list (#10551)
+ - web: fix needed because recent upgrade to task breaks spinner button (#10142)
+ - web: fix package lock out of sync (#10314)
+ - web: fix theme not applying to document correctly (#10721)
+ - web: fixed missed internationalized strings (#10323)
+ - web: lint package-lock.json file (#10157)
+ - web: lintpicking (#10212)
+ - web: move to wireit as the build runner language (#10440)
+ - web: provide 'show password' button (#10337)
+ - web: provide a test framework (#9681)
+ - web: provide better feedback on Application Library page about search results (#9386)
+ - web: provide default endpoint api configuration (#10319)
+ - web: replace all occurences of the theme placeholder (#10749)
+ - web: replace handmade list in Admin Overview with generator, storybook generator, fix storybook, fix bug in list's parent component (#9726)
+ - web: replace multi-select with dual-select for all propertyMapping invocations (#9359)
+ - web: restore hasLaunchUrl to client-side criteria for filtering apps (#10291)
+ - web: search select with focus, autocomplete, and progressive search (#10728)
+ - web: set noopener and noreferrer on all external links (#10304)
+ - web: set noopener and noreferrer on all external links (#10304)
+ - web: update to ESLint 9 (#10812)
+ - web/admin: fix access token list calling wrong API (#10434)
+ - web/admin: fix authentication/enrollment flow in sources being marked as required (#10911)
+ - web/admin: fix missing SAML Provider ECDSA options (#10612)
+ - web/admin: fix selectable card colour in dark theme (#10794)
+ - web/admin: refactor property mappings forms (#10810)
+ - web/admin: show matching user reputation scores in user details (#10276)
+ - web/admin: widen prompt form (#10615)
+ - web/elements: fix empty enterprise banner (#10882)
+ - web/flows: clean up loading, syntax and transitions (#10792)
+ - web/flows: don't grab focus for password input on identification stage (#10593)
+ - web/flows: remove background image link (#10318)
+ - web/flows: remove continue button from AutoSubmit stage (#10253)
+ - web/flows: Simplified flow executor (#10296)
+ - web/flows: Simplified flow executor (#10296)
```

Comment:
weird duplication

Reply:
yeah this is partially caused by how we generate that list, comparing `main` to `version-2024.6` which has some commits cherry picked on it